scanner/decisions/0010-separate-versioning-for-vuln-bundles.md

```diff
@@ -0,0 +1,82 @@
+# 0010 - Separate Versioning Stream for Vulnerability Bundles
+
+- **Author(s):** J. Victor Martins <jvdm@sdf.org>
+- **Created:** [2024-07-22 Mon]
+
+## Status
+
+Status: Accepted
+
+Updates: [#0009](0009-scannerv4-read-manual-vulnerabilities.md)
+
+## Context
+
+A Vulnerability Bundle is an archive containing updates to vulnerability data in Scanner.  They are constantly updated by the Vulnerability Exporter (aka. Exporter), which runs periodically in GitHub's Workflows and uses ClairCore to fetch the security data to convert it to a JSON-based schema.  Then, the workflow bundles the data into a ZIP archive.  The schema is owned by ClairCore.
+
+A Scanner instance runs the Vulnerability Updater (aka. Updater) to fetch the bundle and use ClairCore to parse the schema and update the vulnerability database.  Suppose ClairCore changes the schema in a backward-incompatible manner, and the Exporter updates the bundle.  In that case, existing Scanner instances cannot consume the data.  A Scanner version bump to a version with a ClairCore version supporting the new schema would be required.  Until that bump happens, no vulnerability update will occur. 
```

Comment:
> Suppose ClairCore changes the schema in a backward-incompatible manner, and the Exporter updates the bundle.

Personally I think "suppose" is a bit odd in an ADR because that's a case we know will likely happen. I would say "One case that needs to be taken into consideration is that ClairCore changes the schema in a backward-incompatible manner."

Reply:
Yes, very minor.